client/src/containers/CreateCardFormContainer/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import CreateCardForm from 'src/components/CreateCardForm'
+
+// import styles from './styles.css';
+
+class CreateCardFormContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      description: '',
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
```

Comment:
We can go with autobinding for that: https://medium.com/komenco/react-autobinding-2261a1092849

Can be used:
1) autobind decorator
```js
import autobind from 'autobind-decorator'

@autobind
handleChange(event) {
  this.setState({ description: event.target.value })
}
```
2) arrow function

```js
handleChange = (event) => {
  this.setState({ description: event.target.value })
}
```

instead of:
```js
this.handleChange = this.handleChange.bind(this)
```
in constructor